### PR TITLE
Adding pmd_beamphysics

### PR DESCRIPTION
**Pull request by ChristopherMayes**
Checklist
- [X ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X ] Source is from official source
- [X ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X ] If static libraries are linked in, the license of the static library is packaged.
- [X ] Build number is 0
- [X ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ChristopherMayes** [comment]
@conda-forge/help-python this is ready for review

**Comment by hhslepicka** [comment]
@conda-forge/help-python this is ready for review

File: recipes/pmd_beamphysics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+    
+  host:
+    - python >=3.6
+    - pip
+    
+  run:
+    - python
```
Comment (npavlovikj) [inline recipes/pmd_beamphysics/meta.yaml]: This should be `python >=3.6` as well.

File: recipes/pmd_beamphysics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  url: https://github.com/ChristopherMayes/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0ba2519dc16eb237cfc21886d49b3f69d9fa832e1901cb184141e987dac21c36
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+
```
Comment (npavlovikj) [inline recipes/pmd_beamphysics/meta.yaml]: Please remove empty line.

File: recipes/pmd_beamphysics/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+    
+  host:
+    - python >=3.6
+    - pip
+
```
Comment (npavlovikj) [inline recipes/pmd_beamphysics/meta.yaml]: Please remove empty line.

**Comment by npavlovikj** [comment]
The recipe LGTM - there few minor things, and we can merge it afterwards.

**Comment by ChristopherMayes** [comment]
@npavlovikj I've addressed your comments. Thank you!

**Comment by npavlovikj** [comment]
Thanks for contributing @ChristopherMayes 🎉

**Pull request merged**